### ISSUE TITLE
error[E0425]: cannot find value `m_bIsScoped` in module `cs2dumper::client::C_CSPlayerPawnBase`

### ISSUE THREAD

**Issue by Coforgpaper** [a2x/cs2-dumper#138]
error[E0425]: cannot find value `m_bIsScoped` in module `cs2dumper::client::C_CSPlayerPawnBase`
   --> src/dma/context/mod.rs:106:77
    |
106 |             batcher.read_into(pawn + cs2dumper::client::C_CSPlayerPawnBase::m_bIsScoped, &mut is_scoped);
    |                                                                             ^^^^^^^^^^^ not found in `cs2dumper::client::C_CSPlayerPawnBase`
    |
help: consider importing this constant
    |
1   + use crate::dma::cs2dumper::client::C_CSPlayerPawn::m_bIsScoped;
    |
help: if you import `m_bIsScoped`, refer to it directly
    |
106 -             batcher.read_into(pawn + cs2dumper::client::C_CSPlayerPawnBase::m_bIsScoped, &mut is_scoped);
106 +             batcher.read_into(pawn + m_bIsScoped, &mut is_scoped);
    |

For more information about this error, try `rustc --explain E0425`.
The following warnings were emitted during compilation:

warning: radarflow@0.2.3: no suitable 'git' command found!
warning: radarflow@0.2.3: VERGEN_GIT_COMMIT_DATE set to default
warning: radarflow@0.2.3: VERGEN_GIT_SHA set to default

error: could not compile `radarflow` (bin "radarflow") due to previous error

**Comment by M3351AN** [comment]
cs2_dumper::schemas::client_dll::C_CSPlayerPawn::m_bIsScoped

**Comment by Coforgpaper** [comment]
> cs2_dumper：：schemas：：client_dll：：C_CSPlayerPawn：：m_bIsScoped

?

**Comment by M3351AN** [comment]
cannot find value m_bIsScoped in module cs2dumper::client::C_CSPlayerPawnBase

because

m_bIsScoped is in module cs2dumper::client::C_CSPlayerPawn


> > cs2_dumper：：schemas：：client_dll：：C_CSPlayerPawn：：m_bIsScoped
> 
> ?

**Comment by arrayffs** [comment]
tldr: C_CSPlayerPawnBase was merged / renamed into C_CSPlayerPawn
